src/api-reference/Notifications/PartnerNotifications.markdown

```diff
@@ -0,0 +1,166 @@
+
+### Scopes
+
+`notifications.messages.writeonly` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Request
+
+#### URI
+
+##### Template
+
+```shell
+https://us.api.concursolutions.com/notifications/v1/messages/me/session
+```
+
+##### Parameters
+
+#### Headers
+
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2)
+* [RFC 7231 User-Agent](https://tools.ietf.org/html/rfc7231#section-5.5.3)
+
+#### Payload
+
+```json
+{
+    "sessionId": "D5B80C53-A4D2-4949-8462-D41655F246E2",
+    "message": "template-name",
+    "context": {
+      "url": "https://www.example.com/foo"
+    }
+}
+```
+
+* [Request Payload](#request-schema)
+
+### Response
+
+`OK`
+
+#### Status Codes
+
+* [200 OK](https://tools.ietf.org/html/rfc7231#section-6.3.1)
+* [400 Bad Request](https://tools.ietf.org/html/rfc7231#section-6.5.1)
+* [401 Unauthorized](https://tools.ietf.org/html/rfc7235#section-3.1)
+* [403 Forbidden](https://tools.ietf.org/html/rfc7231#section-6.5.3)
+* [404 Not Found](https://tools.ietf.org/html/rfc7231#section-6.5.4)
+* [500 Internal Server Error](https://tools.ietf.org/html/rfc7231#section-6.6.1)
+* [503 Service Unavailable](https://tools.ietf.org/html/rfc7231#section-6.6.4)
+* [504 Gateway Timeout](https://tools.ietf.org/html/rfc7231#section-6.6.5)
+
+#### Headers
+
+* `concur-correlationid` is a Concur specific custom header used for technical support in the form of a [RFC 4122 A Universally Unique IDentifier (UUID) URN Namespace](https://tools.ietf.org/html/rfc4122)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Content-Encoding](https://tools.ietf.org/html/rfc7231#section-3.1.2.2)
+* [RFC 7230 Content-Length](https://tools.ietf.org/html/rfc7230#section-3.3.2)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+
+#### Payload
+
+* [Request Payload](#request-schema)
+* [Error](#schema-error)
+
+### Example
+
+#### Request
+
+```shell
+POST https://us.api.concursolutions.com/notifications/v1/messages/me/session
+Accept: application/json
+Authorization: Bearer {JWT}
+```
+
+```json
+{
+    "sessionId": "D5B80C53-A4D2-4949-8462-D41655F246E2",
+    "message": "template-name",
+    "context": {
+      "url": "https://www.example.com/foo"
+    }
+}
+```
+
+#### Response
+
+```shell
+HTTP/1.1 200 OK
+Cache-Control: max-age=604800
+Content-Type: application/json
+Date: Wed, 06 Jul 2020 17:33:03 GMT
+Etag: "359670651"
+Expires: Wed, 13 Jul 2020 17:33:03 GMT
+Last-Modified: Fri, 09 Aug 2020 23:54:35 GMT
+Content-Length: 1270
+```
+
+```json
+OK
+```
+
+## <a name="schema"></a>Schema
+
+### <a name="request-schema"></a>Request Payload
+
+Name|Type|Description
+---|---|---
+`sessionId`|`string`|**Required** The unique ID of the session
+`message`|`string`|**Required** The template identifier of the message
+`context.url`|`string`|The context URL to apply to the template
```

Comment:
The reason why we would like to use context.url is because in this case the url is embedded into json context object. In the future we will be introducing new attributes to the json context object.